libraries/classes/FieldMetadata.php

```diff
@@ -233,7 +233,13 @@ public function __construct(int $fieldType, int $fieldFlags, object $field)
             $this->isNotNull = (bool) ($fieldFlags & MYSQLI_NOT_NULL_FLAG);
             $this->isUnsigned = (bool) ($fieldFlags & MYSQLI_UNSIGNED_FLAG);
             $this->isZerofill = (bool) ($fieldFlags & MYSQLI_ZEROFILL_FLAG);
-            $this->isNumeric = (bool) ($fieldFlags & MYSQLI_NUM_FLAG);
+
+            // as flags 32768 can be NUM_FLAG or GROUP_FLAG
+            // reference: https://www.php.net/manual/en/mysqli-result.fetch-fields.php
+            // so also check field type that not 253 (VARCHAR)
+            $this->isNumeric = (bool) (($fieldFlags & MYSQLI_NUM_FLAG)
+                && $this->isType(self::TYPE_INT));
+
```

Comment:
```suggestion
            $this->isNumeric = $this->isType(self::TYPE_INT));
```

This is "safe" definition of numeric because all integers are numeric by definition. `YEAR` type is zero-padded so I wouldn't consider it numeric. DECIMAL should be treated as a string and it seems phpMyAdmin makes no distinction between DECIMAL and FLOAT, which is probably for the best. N.B. the result of `SUM` or `AVG` is always DECIMAL